example_experiments/transfer/asreader_joint_wdw_sciq_train_sciq_test.json

```diff
@@ -0,0 +1,30 @@
+// train a model on a combined dataset with SciQ train and a subset of WhoDidWhat train and see how well it does on SciQ.
```

Comment:
I don't know that there's a whole lot of value in having json files in `example_experiments` that are just one-line changes from other examples.  The scala experiment code is the right place to put all of the experiments that you actually run; this is just for examples of how to run various models.

Also, just a note for something I've been thinking about: I think we should split this repo into two: one for the python code, which we can then set up as a typical python library, and put it on pypi, and so on, and one for the scala experiment code that uses the python library.  I think that'd make everything cleaner and easier to document and understand.  Not sure when we'll actually get around to doing this, but it's on my mind.